NAS2D/Filesystem.cpp

```diff
@@ -270,7 +270,8 @@ void Filesystem::makeDirectory(const std::string& path)
  */
 bool Filesystem::exists(const std::string& path) const
 {
-	return PHYSFS_exists(path.c_str()) != 0;
+	const auto& filePath = findFirstPath(path, mSearchPaths);
```

Comment:
Use `std::filesystem::exists` here.

```
const auto filePath = std::filesystem::path{path}; //This variable is for readability only and can be omitted.
return std::filesystem::exists(filePath);
```

Reply:
Since `fileFirstPath` returns a string it should probably be converted to a path first:

```
const auto filePath = std::filesystem::path{findFirstPath(path, mSearchPaths)}; //This variable is for readability only and can be omitted.
return std::filesystem::exists(filePath);
```

Reply:
The `findFirstPath` function already uses `std::filesystem::exists` internally. No need to call the same function twice. And the `findFirstPath` function is needed, since you need to check a specified search path, which you don't get when calling `std::filesystem::exists` directly.